add instructions for mandatory openshift ingress controller settings (#478)
astronomer/docs: software/third-party-ingress-controllers.md

kubectl label namespace/<ingress namespace> network.openshift.io/policy-group=ingress
```

> For more information, see the [Openshift Network Policy documentation](https://docs.openshift.com/container-platform/4.1/networking/configuring-networkpolicy.html).

Reviewer: ```suggestion
For more information, see the [OpenShift documentation](https://docs.openshift.com/container-platform/4.1/networking/configuring-networkpolicy.html) on configuring network policy.
```

> Additionally, OpenShift's default route admission policy must be configured to allow traffic from multiple namespaces. To do so, run the following command:

Reviewer: ```suggestion
To allow traffic from multiple namespaces, you must also configure OpenShift's default route admission policy. To do so, run the following command:
```

```

> See [Openshift's Network Policy documentation](https://docs.openshift.com/container-platform/4.1/networking/configuring-networkpolicy.html) for more details.

Reviewer 2: ```suggestion
For more information, see the [Openshift Network Policy documentation](https://docs.openshift.com/container-platform/4.1/networking/configuring-networkpolicy.html).
```

> OpenShift's default route admission policy must be configured to allow traffic from multiple namespaces.

Reviewer 2: ```suggestion
Additionally, OpenShift's default route admission policy must be configured to allow traffic from multiple namespaces. You can complete this setup using the following command: 
```

```sh
kubectl -n openshift-ingress-operator patch ingresscontroller/default --patch '{"spec":{"routeAdmission":{"namespaceOwnership":"InterNamespaceAllowed"}}}' --type=merge
```

> See [Openshift's Ingress Operator documentation](https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html) for more details and security-implications for multi-tenant clusters.

Reviewer 2: ```suggestion
For more information about security implications for multi-tenant clusters, see the [Openshift Ingress Operator documentation](https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html).
```
